etc/deploy/aws.sh

```diff
@@ -110,53 +148,44 @@ check_all_nodes_ready() {
     fi
 
     master=`cat nodes.txt | grep master | wc -l`
-    if [ $master != "1" ]; then
+    if [ ${master} != "1" ]; then
         echo "no master nodes found"
         return 1
     fi
 
-    NUM_NODES=3
-    TOTAL_NODES=$(($NUM_NODES+1))
+    num_slaves=3
```

Comment:
I think this should still be `NUM_NODES` so that we define that in a single place (its defined above). I just ran into this myself actually ... I changed `NUM_NODES` but here in the script the ready function expected the wrong number.